src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -527,6 +542,7 @@ protected function getMediaData($media)
                     $fileName = $fileVersion->getName();
                     $storageOptions = $fileVersion->getStorageOptions();
```

Comment:
Why not do the `json_decode` of the storage options here? Then the decodes in the storages could be omitted.

 It seems strange to pass the encoded json string around instead of a clean php array or `\stdClass`.